templates/default/github_pr_comment_trigger.rb.erb

```diff
@@ -0,0 +1,126 @@
+#!/opt/chef/embedded/bin/ruby
+
+# Takes a JSON payload from a Github PR issue webhook from stdin, reads the
+# comment, and if it matches `!bump (major|minor|patch)`, merges the PR, bumps
+# the Chef cookbook's metadata.rb, creates a version tag, and pushes up.
+#
+# Additionally, will pass environments to bump to the environment bumper job if
+# a comma-delimited list of environments (or a * for all of them) is appended
+# to the !bump command, e.g.
+#   !bump patch *
+#   !bump minor production
+#   !bump major dev,staging
+
+require 'git'
+require 'json'
+require 'octokit'
+require 'pp' # TODO: Remove pp after done debugging
+
+METADATA_FILE = 'metadata.rb'.freeze
+CHANGELOG_FILE = 'CHANGELOG.md'.freeze
+COMMAND = '!bump'.freeze
+LEVELS = {
+  'major' => 0,
+  'minor' => 1,
+  'patch' => 2
+}.freeze
+
+# Given a version string of the form 'x.x.x' and a level 0, 1, or 2, increments
+# the specified level and returns the new version string.
+def inc_version(v, level)
+  v = v.split('.')
+  v[level] = v[level].to_i.next.to_s
+  v.join('.')
+end
+
+d = JSON.load(STDIN.read)
+
+# Check if we got a valid request and get the bump level
+comment = d.fetch('comment', {}).fetch('body', '')
+match = comment.match(/^#{COMMAND} (#{LEVELS.keys.join('|')})( \S+(,\S+)*)?$/)
+if match.nil?
+  $stderr.puts '<%= @non_bump_message %>'
+  exit 0
+end
+level = match[1]
+envs = match[2].strip()
+
+# Make sure the issue is a PR
+unless d['issue'].key?('pull_request')
+  $stderr.puts 'Error: Cannot merge issue; can only merge PRs.'
+  exit 1
```

Comment:
the `abort` command does this but in one line.
```
abort 'Error: Cannot merge issues; can only merge PRs.'
```
http://ruby-doc.org/core-2.3.0/Kernel.html#method-i-abort